### DISCUSSION TITLE
插件url失效：Could not GET 'https://maven.kr328.app/releases/com/github/kr328/golang/gradle-plugin/1.0.4/gradle-plugin-1.0.4.pom'. Received status code 521 from server:

### DISCUSSION THREAD

**Discussion by 2691781658** [MetaCubeX/ClashMetaForAndroid#217] [Q&A]
不知道应该怎么办：
library("build-golang", "com.github.kr328.golang:gradle-plugin:$golang") 
library("kaidl-compiler", "com.github.kr328.kaidl:kaidl:$kaidl")
library("kaidl-runtime", "com.github.kr328.kaidl:kaidl-runtime:$kaidl")

**Comment by Vi-Kin** [comment]
解决了吗
